src/ape_compile/__init__.py

```diff
@@ -1,16 +1,12 @@
-from typing import Any, List, Optional
+from typing import List
 
-from pydantic import Field, validator
+from pydantic import validator
 
 from ape import plugins
 from ape.api import PluginConfig
-from ape.logging import logger
 
 
 class Config(PluginConfig):
-    # TODO: Remove in 0.7
-    evm_version: Optional[Any] = Field(None, exclude=True, repr=False)
```

Comment:
Also communicate this one

Reply:
They will all be communicated, see the PR notes.

All the big releases, I make sure to call out the breaking changes in a whole new section... I manually edit these.. These are worth the attention.

Having a bunch of annoying small changes in 1 PR greatly assists with the process though, as less re-testing of all the plugins is required.

Reply:
oh i see this one is missing from the notes, lol, ill add it, thanks!

Reply:
Added